tests/bdd/features/config_file.feature

```diff
@@ -106,3 +106,8 @@ Feature: Multiple journals
         And we use the config "basic_onefile.yaml"
         When we run "jrnl --cf empty_file.yaml"
         Then the error output should contain "Unable to parse config file"
+
```

Comment:
Could you add a test for a config for format "fancy" and resolve the issue? I'm getting this message:
```
┏━ Error ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃  TypeError                                           ┃
┃  unsupported operand type(s) for -: 'str' and 'int'  ┃
┃                                                      ┃
┃  This is probably a bug. Please file an issue at:    ┃
┃  https://github.com/jrnl-org/jrnl/issues/new/choose  ┃
┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
```

Reply:
Thanks for the find. I created a bdd test and fix to resolve the error.